packages/changesets-renovate/src/__tests__/generate-changeset.ts

```diff
@@ -117,6 +117,54 @@ describe('generate changeset file', () => {
     expect(push).toHaveBeenCalledTimes(1)
   })
 
+  it('should generate changeset file, but skip commit and push', async () => {
```

Comment:
do you generate snapshots for this test @wrighbr  ?

Reply:
@philibea I've added the snapshot